crowbar_framework/config/locales/ipmi/en.yml

```diff
@@ -23,6 +23,7 @@ en:
         bmc_user: 'BMC User'
         bmc_password: 'BMC Password'
         bmc_interface: 'BMC Interface'
+        bmc_reconfigure: 'Modify BMC Settings'
```

Comment:
This should be made more specific to mention that it's only about networking. AFAICT, we still modify the user/password.

Reply:
The plan is to make it really 'read-only': https://github.com/skazi0/crowbar-core/blob/6ceee2664d9d7a0777e31d5b90552194bc418c20/chef/cookbooks/ipmi/recipes/ipmi-configure.rb#L118

Reply:
Oh, missed that. Can you add a hint in the UI that it is required that all servers have the same user/password for the BMC?

Reply:
done (not tested yet)